packages/backend/src/index.ts

```diff
@@ -22,6 +22,13 @@
  * Happy hacking!
  */
 
+/**
+ * Necessary global, *strictly singleton*, import of a metadata container. Should be the first item imported of the app.
+ *
+ * This is needed by both inversify and nest.js.
+ */
+import 'reflect-metadata';
```

Comment:
Btw, this is only needed when using decorators right?

Reply:
correct yeah